web/src/app/administration/components/workbasket-distribution-targets-list/workbasket-distribution-targets-list.component.scss

```diff
@@ -12,7 +12,7 @@
     margin-left: 0.5rem;
   }
 
-  &__list {
+  &__list-without-filter {
```

Comment:
for mods you should use --, for example list--no-filter and list--with-filter 

see modifier: http://getbem.com/introduction/